### PR TITLE
Clean up console logs and commented code.

### PR DESCRIPTION
**Pull request by elsafluss**
### Is this a fix or a feature?
fix
### What is the change/fix?
cleaning up spacing and console logs
### Where should the reviewer start?
top of any file
### How should this be tested?
start up the servers and open the page

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -127,18 +124,6 @@ function cardButtonConditionals(event) {
     favoriteCard(event, cookbook)
   } else if (event.target.classList.contains("card-picture")) {
     displayDirections(event)
-    // } else if (event.target.classList.contains("add-button")) {
```
Comment (DrewBradley) [inline src/scripts.js:130]: nice

**Review by colefiscus — APPROVED** [review]
I approve this, vehemently.

File: src/scripts.js [hunk 2]
```diff
@@ -217,25 +201,24 @@ function checkFields() {
...
 
-function postIngredient() { 
+function postIngredient() {
   getIngredients()
-  .then((ingredientsData) => {
-    let ingName = document.querySelector(".ingredient-name").value;
-    let ingId = ingredientsData.find(ingredient => {
-      if (ingredient.name) {
-      return ingredient.name === ingName
+    .then((ingredientsData) => {
```
Comment (DrewBradley) [inline src/scripts.js:206]: so fresh an so clean clean

**Review by DrewBradley — APPROVED** [review]
Amazing work Elsa!

**Pull request merged**